src/platforms/linux/daemon/dbusservice.h

```diff
@@ -24,7 +25,11 @@ class DBusService final : public Daemon, protected QDBusContext {
   DBusService(QObject* parent);
   ~DBusService();
 
+  enum AppState { Active, Excluded, Blocked };
```

Comment:
Is this 'Blocked' state used?

Reply:
Not anymore. In the previous iteration of the Linux split tunneling feature, we had the ability to deny network access to applications entirely with this feature, and that's what this state did. But sometime last year we needed to migrate the split tunneling feature to cgroupsv2 and we lost that feature in the process but left it in the API.

This state is a vestigial reminder of that old code.

Reply:
I have removed this state.